build.gradle

```diff
@@ -87,7 +87,7 @@ repositories {
 if (JavaVersion.current() <= JavaVersion.VERSION_11) {
```

Comment:
just out of curiosity: why did you put `VERSION_11` instead of `VERSION_1_8`? Optionals were introduced in 1.8, right?